CMakeLists.txt

```diff
@@ -37,7 +37,7 @@ set(AMR_WIND_PRECISION "DOUBLE" CACHE STRING "Floating point precision SINGLE or
 option(AMR_WIND_USE_INTERNAL_AMREX "Add AMReX as subproject" ON)
 option(AMR_WIND_USE_INTERNAL_AMREX_HYDRO "Add AMReX-Hydro as subproject" ON)
 option(AMR_WIND_USE_INTERNAL_WAVES2AMR "Add Waves2AMR as subproject" ON)
-option(AMR_WIND_ENABLE_HDF5 "Enable HDF5 library" OFF)
+option(AMR_WIND_ENABLE_HDF5 "Enable HDF5 library" ON)
```

Comment:
This is changing the default requirements to include HDF5 for compiling AMR-Wind. What is the intent behind this change?